packages/core/src/ref-utils.ts

```diff
@@ -12,6 +12,10 @@ export function isRef(node: any): node is OasRef {
   return node && typeof node.$ref === 'string';
 }
 
+export function isRelativeReference(value: any): boolean {
```

Comment:
Could you use the [existing check](https://github.com/Redocly/redocly-cli/blob/1f0ca52ce083f21bc1b39ec4f5da8a18ece5daf1/packages/core/src/ref-utils.ts#L69) instead?